### PR TITLE
Fix chat in standalone notes

### PR DESCRIPTION
**Pull request by ComputelessComputer**
Mount the shared chat panels in standalone note windows so Ask Anarlog becomes visible after opening it. Add regression coverage for the route layout.

Verification:
- pnpm -F @hypr/desktop exec vitest run "src/routes/app/-note.$sessionId.test.tsx"
- pnpm -F @hypr/desktop typecheck
- pnpm exec dprint check

---

> [!NOTE]
> **Medium Risk**
> Touches window resize/expansion logic and global Escape handling reused across main and standalone windows; behavior is well covered by new tests but layout edge cases could still differ from the main shell.
> 
> **Overview**
> Standalone note windows now mount **`MainChatPanels`** around the note so floating and docked chat (Ask Anarlog) actually renders, with a dedicated **`autoSaveId`**, **no left sidebar**, and a **420px** note minimum instead of the main app’s 500px + sidebar rules.
> 
> **`MainChatPanels`** accepts optional **`leftSidebarAvailable`**, **`noteSurfaceMinWidth`**, and **`autoSaveId`**; width guards and min-body math use those values, and visible-width measurement prefers **`data-main-chat-panel-group`** so Tauri window expansion works outside the main shell.
> 
> Escape in standalone notes matches the main app: shared **`getEscapeShortcutContext`** / **`shouldSkipEscapeShortcut`**, deferred handling—**close chat first**, then close the window—and respect overlays that consume Escape. Changelog notes chat from standalone note windows; tests cover layout, escape order, and standalone panel sizing.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit a28fd3cf440b74dc1f88714b108fcb7af8f09452. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

File: apps/desktop/src/routes/app/note.$sessionId.tsx [hunk 1]
```diff
@@ -84,19 +100,26 @@ export function useStandaloneNoteTab(sessionId: string) {
...
+
       void getCurrentWindow().close();
     };
 
     window.addEventListener("keydown", handleKeyDown, { capture: true });
     return () => {
       window.removeEventListener("keydown", handleKeyDown, { capture: true });
     };
-  }, []);
+  }, [chat.mode, chat.sendEvent]);
```
[the remark this answers is not in the document]
Comment (ComputelessComputer) [reply apps/desktop/src/routes/app/note.$sessionId.tsx:136]: Fixed in 90b52603a6: standalone Escape handling now defers and reuses the main-window escape-consumer decision logic, so overlays can prevent the event before chat or the window closes. Added a regression test for an overlay consuming Escape; focused shortcut/route/panel tests pass 45/45.

File: apps/desktop/src/shared/main/chat-panels.tsx [hunk 2]
```diff
@@ -49,6 +60,7 @@ export function MainChatPanels({ children }: { children: React.ReactNode }) {
         <>
           <ResizablePanelGroup
             autoSaveId="main-chat"
+            data-main-chat-panel-group
```
[the remark this answers is not in the document]
Comment (ComputelessComputer) [reply apps/desktop/src/shared/main/chat-panels.tsx:65]: Fixed in a28fd3cf44: MainChatPanels now accepts an explicit autoSaveId, standalone notes use standalone-note-chat, and regression coverage verifies main and standalone layouts persist under separate keys.
